packages/react-native-editor/ios/Podfile

```diff
@@ -54,20 +54,13 @@ target 'GutenbergDemo' do
   end
 
   post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
     react_native_post_install(
       installer,
       config[:reactNativePath],
-      # Set `mac_catalyst_enabled` to `true` in order to apply patches
-      # necessary for Mac Catalyst builds
       :mac_catalyst_enabled => false
     )
 
-    # The following workaround is needed to avoid the error `typedef redefinition with different types ('uint8_t' (aka 'unsigned char') vs 'enum clockid_t')`.
-    # This solution is referenced in https://github.com/facebook/react-native/issues/39568#issuecomment-1762890606.
-    # It will be needed until RCT-Folly version is bumped in React Native to version v2022.08.29.00 or above.
-    # Referece: https://github.com/facebook/folly/commit/4a2410fae65afb85e1fec6d922005054b05de59f
-    __apply_Xcode_12_5_M1_post_install_workaround(installer)
```

Comment:
This through an error during `pod install` and appears to no longer be necessary.